pubspec.yaml

```diff
@@ -6,7 +6,7 @@ author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/html
 
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.2.2 <3.0.0'
```

Comment:
Just make this `>=2.3.0...`

Reply:
Done.

Reply:
@nshahan I've also updated the version in .travis.yml.